Diamas Project/client/data/source/root/data/map_config.txt
List NoSnowMaps
{
	green_screen
}

List NoBoomMaps
{
	green_screen
}

List NoHorseSkillMaps
{
	metin2_map_a1
	metin2_map_a3
	metin2_map_b1
	metin2_map_b3
	metin2_map_c1
	metin2_map_c3
}

List DuelMaps
{
	160
	175
	229
	223
	320
	234
}

List MapIndex
{
	1	metin2_map_a1	enter_the_east.mp3
	3	metin2_map_a3	back_to_back.mp3
	21	metin2_map_b1	enter_the_east.mp3
	23	metin2_map_b3	back_to_back.mp3
	28	metin2_map_duel2	enter_the_east.mp3
	41	metin2_map_c1	enter_the_east.mp3
	43	metin2_map_c3	back_to_back.mp3
	81	metin2_map_wedding_01	wedding.mp3
	110	metin2_map_t3	last-war2.mp3
	111	metin2_map_t4	last-war2.mp3
	113	metin2_map_oxevent	open_the_gate.mp3
	160	metin2_map_n_desert_01	open_the_gate.mp3
	175	metin2_map_n_desert_01	desert.mp3
	229	gm_guild_build	enter_the_east.mp3
	400	map_a2	enter_the_east.mp3
	500	metin2_map_skipia_dungeon_02	enter_the_east.mp3
}

List MapIndexTranslation
{
	1 MAP_A1
	3 MAP_A3
	21 MAP_B1
	23 MAP_B3
	28	MAP_DUEL2
	41 MAP_C1
	43 MAP_C3
	81 MAP_WEDDING_01
	110 MAP_T3
	111 MAP_T4
	113 MAP_OXEVENT
	160 MAP_DESERT
	175 MAP_DESERT22
	229 MAP_PATHWAY_ICE_02
	400	MAP_N_THREEWAY
	500	MAP_SKIPIA_DUNGEON_02
}


Diamas Project/client/data/source/root/data/wiki_config.txt
List ItemOriginMap
{
	50187	ITEM_STARTER_CHEST
}

List MobOriginMap
{
	719	MAP_OWL
	7191	MAP_OWL
	472	MAP_GARDEN
	4721	MAP_GARDEN
	676	MAP_DEEP_CAVE
	6761	MAP_DEEP_CAVE
	880	MAP_NEVERLAND
	8801	MAP_NEVERLAND
	878	MAP_WOOD
	8781	MAP_WOOD
	9263	MAP_ICESTORM
	9264	MAP_ICESTORM
	689	MAP_SNOW_QUEEN
	879	MAP_ARTEMIS
	964	MAP_AQUA
	470	MAP_SOLITARY
	471	MAP_LDK
	949	MAP_DRAGON_FIRE
	9430	MAP_WORLDBOSS1
	9431	MAP_WORLDBOSS2
	283	MAP_DEMON
	2831	MAP_DEMON
}

List BossChestVnums
{
	38881
	38881
	38886
	38887
	38888
	38889
	38890
	38891
	38892
	50259
	253700
	253701
	253702
	253703
	253710
	253711
	253712
	253713
	253730
	253731
	253732
	253733
	253735
	253736
	253737
	253738
	253739
	253740
	253750
	253751
	253752
	253753
	342110
	342112
}

List EventChestVnums
{
	50259
}


List AltChestVnums
{
	50259
}

List ItemWhiteList
{
	34910
	34920
	34930
	34940
	35900
	35910
	35920
	35930
	35940
	35950
	35960
	35970
	35980
	35990
	36000
	36010
	36020
	36030
	36040
	36050
	36060
	37910
	37920
	37930
	37940
	37950
	37960
	37970
	37980
	37990
	38210
	38220
	38230
	38240
	38250
	36900
	36910
	36920
	36930
	36940
	36950
	36960
	140
	270
	280
	4040
	3190
	2160
	5330
	7190
	150
	1100
	2140
	3130
	5100
	7140
	11290
	11490
	11690
	11890
	12240
	12380
	12520
	12660
	14200
	15200
	16200
	17200
	13060
	180
	190
	1130
	2170
	3160
	5120
	7180
	12220
	12360
	12500
	12640
	14180
	16180
	17180
	13200
	15180
	292050
	292060
	292070
	292080
	8090
	8100
	8110
	8120
	8130
	8140
	8150
	8160
	8170
	8180
	8190
	8200
	8210
	8220
	8230
	8240
	8250
	8260
	8270
	8280
	8290
	8300
	8320
	8330
	8340
	8350
	8360
	8370
	8380
	8390
	8400
	8410
	8420
	8430
	8440
	8450
	8460
	8470
	8480
	8490
	8500
	8510
	8520
	8530
	8540
	8550
	8560
	8570
	8580
	8590
	8600
	8610
	8620
	8630
	8640
	8650
	8660
	8670
	8680
	8690
	8700
	8710
	8720
	8730
	8740
	8750
	8760
	8770
	8780
	8790
	8800
	8810
	8820
	8830
	8840
	8850
	8860
	8870
	8880
	8890
	8900
	8910
	8920
	8930
	8940
	9640
	9650
	9660
	9670
	12010
	12020
	12030
	12040
	55100
	55110
	55120
	55130
	55140
	55150
	55160
	55170
	55180
	55190
	55200
	55210
	291370
	291390
	291410
	291430
	291380
	291400
	291420
	291440
	55380
	55390
	55500
	55510
	55700
	55710
	55720
	55730
	9680
	9690
	9700
	9710
	291340
	291360
	55370
	55690
	9720
	291530
	291540
	55450
	55770
	9760
	291470
	291480
	55420
	55740
	9730
	291490
	291500
	55430
	55750
	9740
	18010
	18020
	18070
	9910
	9920
	18080
	18030
	18040
	18050
	18060
	18090
	291510
	291520
	55440
	55760
	9750
	220
	210
	3200
	4030
	2190
	5130
	7170
	10140
	10150
	13130
	11100
	12140
	57140
	15090
	20140
	23130
	31120
	22140
	20150
	27140
	25090
	20200
	33130
	33100
	32140
	40050
	37140
	35090
	40140
	33140
	31110
	32150
	40150
	47140
	55090
	8980
	8990
	9510
	9520
	9530
	9540
	72800
	72810
	72820
	72830
	72840
	72850
}

List MobWhitelist
{
	271
	272
	273
	274
	275
	276
	277
	278
	279
	280
	281
	282
	283
	2831
	8408
	9430
	9431
	719
	7191
	472
	4721
	676
	6761
	880
	8801
	878
	8781
	9263
	9264
	8424
	689
	8407
	879
	8421
	964
	8433
	470
	8406
	471
	8302
	949
	8424
	710
	711
	713
	463
	464
	465
	959
	960
	961
	860
	861
	866
	460
	461
	467
	941
	942
	945
	712
	714
	715
	716
	717
	718
	462
	466
	473
	474
	475
	476
	477
	478
	479
	480
	481
	469
	670
	671
	672
	673
	674
	675
	9330
	9331
	9332
	9333
	870
	871
	873
	862
	863
	864
	867
	868
	869
	875
	874
	872
	876
	8750
	8751
	8752
	8753
	4139
	4140
	4141
	4142
	4143
	4144
	4145
	4146
	4147
	4148
	4149
	4150
	4151
	4152
	4153
	4154
	4155
	4156
	4157
	4158
	4159
	8472
	8473
	8474
	8475
	8476
	24361
	24362
	24363
	24364
	24365
	24366
	24367
	24368
	24369
	24370
	24371
	24372
	24373
	24374
	24375
	24376
	24377
	24378
	24379
	24380
	24381
	24382
	24383
	24384
	24385
	24386
	24387
	24388
	24389
	24482
	24483
	24484
	24485
	24486
	24487
	24488
	28494
	28495
	28496
	28497
	28498
	28499
	4399
	4400
	4401
	4402
	4403
	4404
	4405
	4406
	4407
	4408
	55022
	8702
	8703
	220
	219
	218
	55013
	55014
	55015
	221
	222
	223
	224
	225
	226
	227
	228
	207
	208
	209
	55016
	55018
	210
	211
	212
	213
	214
	215
	216
	217
	4409
	4410
	55000
	55001
	55002
	55019
	55021
	55003
	55004
	55005
	55006
	55007
	55008
	55009
	55010
	55011
	55012
}

List CostumeWeaponVnum	
{	
	411109
	411119
	411129
	411139
	411149
	411159
	411169
	421509
	421519
	421529
	421539
	421549
	421559
	421569
	411509
	411519
	411529
	411539
	411549
	411559
	411569
	411709
	411719
	411729
	411739
	411749
	411759
	411769
	410909
	410919
	410929
	410939
	410949
	410959
	410969
	42409
	42419
	42429
	42439
	42449
	42459
	42469
	411609
	411619
	411629
	411639
	411649
	411659
	411669
	411309
	411319
	411329
	411339
	411349
	411359
	411369
	40129
	40130
	40131
	40132
	40133
	40134
	40108
	40109
	40110
	40111
	40112
	40113
	48649
	48659
	48669
	48679
	48689
	48699
	48709
	100029
	10003019
	10002019
	10004009
	10007019
	100039
	10003029
	10002029
	10001019
	10005009
	10007029
	100049
	10003039
	10002039
	10004019
	10005019
	10007039
	100059
	10003049
	10002049
	10001029
	10005029
	10007049
	100069
	10003059
	10002059
	10004029
	10007059
	1000299
	10003219
	10002159
	10001179
	10005119
	10007169
	100079
	10003069
	10002069
	10001039
	10005039
	10007069
	100089
	10003079
	10002079
	10001049
	10005049
	10007079
	100099
	10003089
	10002089
	10001059
	10005059
	10007089
	1000109
	10003099
	10002099
	10001069
	10007099
	1000119
	10003109
	10002109
	10001079
	10005069
	10007109
	1000129
	10003119
	10002119
	10001089
	10005079
	10007119
	1000139
	10003129
	10002129
	10001099
	10005089
	10007129
	1000169
	1000249
	10003149
	10002139
	10001119
	10005099
	10007139
	1000179
	1000259
	10003159
	10002189
	10001129
	10007159
	1000209
	1000469
	1000479
	10001349
	10002379
	10005349
	10007379
	1000309
	10003229
	10002209
	10001189
	10005169
	10007309
	1000509
	10003509
	10002509
	10001509
	10005509
	10007509
	721579
	721589
	721599
	721609
	721619
	721629
	721639
	42709
	42719
	42729
	42739
	42749
	42759
	42769
	42809
	42819
	42839
	42849
	42859
	42869
	42909
	42919
	42939
	42949
	42959
	42969
	43309
	43319
	43329
	43339
	43349
	43359
	43369
	43409
	43419
	43429
	43439
	43449
	43459
	43469
	411209
	411219
	411229
	411239
	411249
	411259
	411269
	56279
	56280
	56281
	56282
	56283
	56284
	56285
	56272
	56273
	56274
	56275
	56276
	56277
	56278
	56265
	56266
	56267
	56268
	56269
	56270
	56271
	42609
	42619
	42629
	42639
	42649
	42659
	42669
	43109
	43119
	43129
	43139
	43149
	43159
	43169
	43209
	43219
	43229
	43239
	43249
	43259
	43269
	510509
	510519
	510529
	510539
	510549
	510559
	510569
	43609
	43619
	43629
	43639
	43649
	43659
	43669
	48579
	48589
	48599
	48609
	48619
	48629
	48639
}

List CostumeArmorVnum
{
	41584
	41585
	41586
	41587
	41117
	41118
	41324
	41325
	41320
	41321
	41496
	41497
	41498
	41499
	41528
	41529
	41530
	41531
	41560
	41561
	41562
	41563
	48359
	48369
	48379
	48389
	511089
	511099
	511109
	511119
	511129
	511139
	511149
	511159
	431009
	431019
	431029
	431039
	431049
	431059
	431069
	431079
	6621780
	6621781
	6621782
	6621783
	41600
	41601
	41602
	41603
	41678
	41680
	41679
	41681
	41901
	41902
	41903
	41003
	41004
	100011219
	100011419
	100011619
	100011819
	100011229
	100011429
	100011629
	100011829
	100011239
	100011439
	100011639
	100011839
	100011249
	100011449
	100011649
	100011849
	100011259
	100011459
	100011659
	100011859
	100011269
	100011469
	100011669
	100011869
	100011279
	100011479
	100011679
	100011879
	100011289
	100011489
	100011689
	100011889
	100012019
	100012029
	100012039
	100012049
	100019299
	100019499
	100019699
	100019899
	41315
	41316
	44729
	44739
	44749
	44759
	44689
	44699
	44709
	44719
	232632
	232633
	232634
	232635
	332009
	332019
	332029
	332039
	44899
	44909
	44919
	44929
	48399
	48409
	48419
	48429
	49359
	49369
	49379
	49389
	431249
	431259
	431269
	431279
	43809
	43819
	43829
	43839
	49959
	49969
	49979
	49989
	43969
	43979
	43989
	43999
	72867
	72868
	72869
	72870
	44049
	44059
	44069
	44079
	514109
	514119
	514129
	514139
	515109
	515119
	515129
	515139
	44009
	44019
	44029
	44039
	43889
	43899
	43909
	43919
	44509
	44519
	44529
	44539
	56214
	56215
	41291
	41292
	41293
	41294
	41295
	41296
	41297
	41298
	41504
	41505
	41506
	41507
	41540
	41541
	41542
	41543
}

List CostumeHairVnum
{
	56209
	56210
	45001
	45002
	45119
	45120
	45121
	45122
	45123
	45124
	45125
	45126
	45190
	45191
	45192
	45193
	45222
	45223
	45224
	45225
}

List CostumeWingVnum
{
	233650
	233653
	234650
	233666
	233651
	233652
	240000
	240001
	240002
	240003
	240004
	240005
	233931
	233932
	234936	
	234937
	234938
	234939
	234940
	234941
}

List CostumeShiningVnum
{
	233741
	233747
	233748
	233780
	233781
	233782
	233764
	233770
	233755
	233769
	233754
	233776
	240006
	240007
	240008
	240009
	240010
	240011
	240012
	240013
	240014
	240015
	240016
	240017
	233720
	233721
	233722
	233723
	233724
	233725
	233926
	233927
	233928
	233923
	233924
	233925
}


Diamas Project/game-database/game/m2_rework_project/m2_rework_project/windows_server/common/data/settings.lua
-- /goto #??
-- add_goto_info("string_type_name", 0, map_name, x, y)
add_goto_info("a1", 0, 1, 					597, 682	)
add_goto_info("a3", 0, 3, 					536, 584	)
add_goto_info("b1", 0, 21, 					557, 555)
add_goto_info("b3", 0, 23, 					361, 301)
add_goto_info("c1", 0, 41, 					480, 736)
add_goto_info("c3", 0, 43, 					539, 378)
add_goto_info("snow", 1, 61, 			758, 137)
add_goto_info("snow", 2, 61, 			168, 213)
add_goto_info("snow", 3, 61, 			1334, 20)
add_goto_info("flame", 1, 62, 			106, 141)
add_goto_info("flame", 2, 62, 			90, 78)
add_goto_info("flame", 3, 62, 			1419, 75)
add_goto_info("desert", 1, 63, 			130, 140)
add_goto_info("desert", 2, 63, 			171, 163)
add_goto_info("desert", 3, 63, 			1392, 16)
add_goto_info("wedding", 0, 81, 		58, 25)
add_goto_info("three", 1, 64, 			1461, 83)
add_goto_info("three", 2, 64, 			144, 743)
add_goto_info("three", 3, 64, 			653, 142)
add_goto_info("milgyo", 1, 65, 			160, 924)
add_goto_info("milgyo", 2, 65, 			160, 924)
add_goto_info("milgyo", 3, 65, 			160, 924)
add_goto_info("skel", 1, 65, 				529, 596)
add_goto_info("skel", 2, 65, 				529, 596)
add_goto_info("skel", 3, 65, 				529, 596)
add_goto_info("209", 0, 209, 8537, 14164)
add_goto_info("210", 0, 210, 8167, 15050)
add_goto_info("212", 0, 212, 3200, 15290)		add_bgm_info(212, "dungeon_fire.mp3", 0.5);			-- metin2_map_n_flame_dragon


add_goto_info("trent", 0 , 67, 71 , 57 )
add_goto_info("trent2",0 , 68, 703 , 708 )
add_goto_info("sungzi",0 , 114, 250, 250 )
add_goto_info("fp1", 0, 115, 82, 235 )
add_goto_info("fp2", 0, 116, 82, 235 )
add_goto_info("fp3", 0, 117, 82, 235 )
add_goto_info("fh1", 0, 118, 250, 800 )
add_goto_info("fh2", 0, 119, 250, 800 )
add_goto_info("fh3", 0, 120, 250, 800 )
add_goto_info("sungzisnow", 0,121, 450, 250 )
add_goto_info("fs1", 0, 122, 225, 655 )
add_goto_info("fs2", 0, 123, 225, 655 )
add_goto_info("fs3", 0, 124, 225, 655 )

add_goto_info("sungzidesert", 0,125, 450, 250 )
add_goto_info("fd1", 0, 126, 225,  655 )
add_goto_info("fd2", 0, 127, 225, 655 )
add_goto_info("fd3", 0, 128, 225, 655 )

add_goto_info("milgyomid", 0 , 65, 529,594 )

add_goto_info("sd3", 0, 217, 82, 78)

add_goto_info("dc1", 0, 216, 81, 63)
add_goto_info("dc2", 0, 216, 552, 46)
add_goto_info("dc3", 0, 216, 1372, 249)
add_goto_info("dc4", 0, 216, 70, 579)
add_goto_info("dc5", 0, 216, 845, 895)
add_goto_info("dc6", 0, 216, 1303, 704)
add_goto_info("dc7", 0, 216, 73, 1156)

add_goto_info("a4", 0, 4, 0, 0)
add_goto_info("b4", 0, 26, 0, 0)
add_goto_info("c4", 0, 46, 0, 0)

add_goto_info("cape", 0, 218, 803, 1245)
add_goto_info("dawn", 0, 302, 487, 171)
add_goto_info("bay", 0, 303, 373, 1454)
add_goto_info("thunder", 0, 304, 81, 1442)

add_goto_info("oxevent", 0, 113, 260, 215)
add_goto_info("em1", 0, 181, 225, 225 )
add_goto_info("em1", 0, 182, 225, 225 )
add_goto_info("em1", 0, 183, 225, 225 )

add_goto_info("heaven", 0, 64, 282, 1449)
add_goto_info("heaven2", 0, 73, 882, 718)
add_goto_info("heaven3", 0, 208, 0, 0)

arena.add_map(541, 128, 105, 128, 149)
arena.add_map(542, 128, 105, 128, 149)
arena.add_map(543, 128, 105, 128, 149)

add_goto_info("OXEVENT", 0, 113, 260, 215)
add_goto_info("gm", 0, 401, 451, 441)

-- /goto GF special
-- /goto GF Monkey Dungeon
add_goto_info("MDred", 0, 5, 72, 130)
add_goto_info("MDyellow", 0, 25, 72, 130)
add_goto_info("MDblue", 0, 45, 72, 130)
add_goto_info("MDmiddle", 0, 108, 72, 132)
add_goto_info("MDexpert", 0, 109, 72, 131)
-- /goto GF Guildzone
add_goto_info("GZred", 0, 4, 75, 49)
add_goto_info("GZyellow", 0, 24, 423, 103)
add_goto_info("GZblue", 0, 44, 411, 131)
-- /goto GF sneakfield
add_goto_info("snake", 0, 69, 94, 94)
-- /goto GF Giant
add_goto_info("giant", 0, 70, 0, 0)
-- /goto GF SD1
add_goto_info("sd1", 0, 104, 88, 106)
-- /goto GF SD2
add_goto_info("sd2", 0, 71, 382, 273)

add_goto_info("flamed", 0, 351, 353, 360)

add_bgm_info( 1, "enter_the_east.mp3", 0.5);
add_bgm_info(21, "enter_the_east.mp3", 0.5);
add_bgm_info(41, "enter_the_east.mp3", 0.5);
add_bgm_info( 3, "back_to_back.mp3", 0.5);
add_bgm_info(23, "back_to_back.mp3", 0.5);
add_bgm_info(43, "back_to_back.mp3", 0.5);
add_bgm_info(63, "open_the_gate.mp3", 0.5);
add_bgm_info(69, "open_the_gate.mp3", 0.5);
add_bgm_info(70, "open_the_gate.mp3", 0.5);
add_bgm_info(67, "a_rhapsody_of_war.mp3", 0.5);
add_bgm_info(68, "lost_my_name.mp3", 0.5);
add_bgm_info(65, "wonderland.mp3", 0.5);
add_bgm_info(61, "mountain_of_death.mp3", 0.5);
add_bgm_info(64, "save_me.mp3", 0.5);
add_bgm_info(74, "mountain_of_death.mp3", 0.5);
add_bgm_info(75, "follow_war_god.mp3", 0.5);
add_bgm_info(76, "mountain_of_death.mp3", 0.5);
add_bgm_info(77, "save_me.mp3", 0.5);
add_bgm_info(78, "wonderland.mp3", 0.5);
add_bgm_info(104, "Only_my_battle.mp3", 0.5);
add_bgm_info(62, "follow_war_god.mp3", 0.5);
add_bgm_info(66, "death_of_landmark.mp3", 0.5);
add_bgm_info(107, "monkey_temple.mp3", 0.5);
add_bgm_info(108, "monkey_temple.mp3", 0.5);
add_bgm_info(109, "monkey_temple.mp3", 0.5);
add_bgm_info(114, "last-war2.mp3", 0.5);
add_bgm_info(115, "last-war2.mp3", 0.5);
add_bgm_info(116, "last-war2.mp3", 0.5);
add_bgm_info(117, "last-war2.mp3", 0.5);
add_bgm_info(118, "last-war2.mp3", 0.5);
add_bgm_info(119, "last-war2.mp3", 0.5);
add_bgm_info(120, "last-war2.mp3", 0.5);
add_bgm_info(121, "last-war2.mp3", 0.5);
add_bgm_info(122, "last-war2.mp3", 0.5);
add_bgm_info(123, "last-war2.mp3", 0.5);
add_bgm_info(124, "last-war2.mp3", 0.5);
add_bgm_info(125, "last-war2.mp3", 0.5);
add_bgm_info(126, "last-war2.mp3", 0.5);
add_bgm_info(127, "last-war2.mp3", 0.5);
add_bgm_info(128, "last-war2.mp3", 0.5);
add_bgm_info(181, "last-war2.mp3", 0.5);
add_bgm_info(182, "last-war2.mp3", 0.5);
add_bgm_info(183, "last-war2.mp3", 0.5);
add_bgm_info(216, "catacomb_of_devil.mp3", 0.5);
add_bgm_info(71, "Only_my_battle.mp3", 0.5);
add_bgm_info(217, "Only_my_battle.mp3", 0.5);
add_bgm_info(301, "another_way.mp3", 0.5);
add_bgm_info(302, "misty_forest.mp3", 0.5);
add_bgm_info(303, "blacksea.mp3", 0.5);
add_bgm_info(304, "mt.mp3", 0.5);
add_bgm_info(209, "enter_the_east.mp3", 0.5);		-- metin2_map_Mt_Th_dungeon_01
add_bgm_info(210, "enter_the_east.mp3", 0.5);		-- metin2_map_dawnmist_dungeon_01
add_bgm_info(210, "plant_dungeon.mp3", 0.5);
add_bgm_info(212, "owl_dungeon.mp3", 0.5);
set_bgm_volume_enable();

dofile("data/BlueDragon.lua")
dofile("data/quest/GFquestlib.lua")



--Goto info
add_goto_info("shadow_enter", 0, 145, 19020, 22690)
add_goto_info("shadow_tower1", 0, 144, 16894, 22609)
add_goto_info("shadow_tower2", 0, 144, 17841, 22987)
add_goto_info("shadow_tower3", 0, 144, 17127, 22671)
add_goto_info("shadow_tower4", 0, 144, 17428, 22644)
add_goto_info("shadow_tower5", 0, 144, 17205, 22927)
add_goto_info("shadow_tower6", 0, 144, 17752, 22714)
add_goto_info("shadow_tower7", 0, 144, 17877, 23699)
add_goto_info("shadow_tower8", 0, 144, 16873, 23787)
add_goto_info("shadow_tower9", 0, 144, 17268, 23855)


--- Bgm (music) info
add_bgm_info(330, "shadow_dungeon.mp3", 0.5);
add_bgm_info(331, "shadow_dungeon.mp3", 0.5);
add_bgm_info(332, "shadow_dungeon.mp3", 0.5);
add_bgm_info(320, "shadow_enter.mp3", 0.5);
add_bgm_info(230, "ancient_pyramid.mp3", 0.5);

